kimchi/src/circuits/lookup/lookups.rs

```diff
@@ -550,3 +550,50 @@ fn lookup_pattern_constants_correct() {
         assert_eq!((pat, pat.max_joint_size()), (pat, max_joint_size as u32));
     }
 }
+
+#[cfg(feature = "wasm_types")]
+pub mod wasm {
+    use super::*;
+
+    #[wasm_bindgen::prelude::wasm_bindgen]
+    impl LookupPatterns {
+        #[wasm_bindgen::prelude::wasm_bindgen]
```

Comment:
Why don't you use constructor?

Reply:
See https://rustwasm.github.io/wasm-bindgen/reference/attributes/on-rust-exports/constructor.html

Reply:
Is it related to this PR: https://github.com/o1-labs/o1js-bindings/pull/146/files?

Reply:
Done